actors/evm/src/interpreter/precompiles/mod.rs

```diff
@@ -45,7 +45,7 @@ impl<RT: Runtime> Precompiles<RT> {
         Some(resolve_address::<RT>),          // 0xfe00..01
         Some(lookup_delegated_address::<RT>), // 0xfe00..02
         Some(call_actor::<RT>),               // 0xfe00..03
-        None,                                 // 0xfe00..04 DISABLED
```

Comment:
Just a heads up the randomness precompile used to be 5 and 4 was [get_actor_type](https://github.com/filecoin-project/builtin-actors/pull/995/files#diff-99cec8914fe93e1f511349a144660b5f5bc1201e3ed0f89d72cd4c5bdbffbce4L60).  I wouldn't expect it to matter but then again we kept this hole in the past so maybe there is some issue worth checking up on.

Reply:
Oh. Hm. I don't think it'll be an issue at this point, but I'm not sure why we disabled one and replaced the other.

Reply:
I don't think it'll be an issue either, but out of precaution I'm going to leave the hole here and use number 6.